Assets/Astralization/Scripts/_Common/DefaultPlayerState.cs

```diff
@@ -0,0 +1,101 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class DefaultPlayerState : PlayerState
+{
+    #region Movement Variables
```

Comment:
More like `Handler Variables` instead of `Movement Variables` because the attribute is not only about movement. wdyt? 
```suggestion
    #region Handler Variables
```

Reply:
i agree, handler is good and more general naming